src/test/java/io/vertx/core/shareddata/AsyncMapTest.java

```diff
@@ -529,6 +542,22 @@ public void testMapReplaceIfPresentClusterSerializableObject() {
     testMapReplaceIfPresent(new SomeClusterSerializableObject("foo"), new SomeClusterSerializableObject("bar"), new SomeClusterSerializableObject("quux"));
   }
 
+  @Test
+  public void testMapReplaceIfPresentTtl() {
+    getVertx().sharedData().<String, String>getAsyncMap("foo", onSuccess(map -> {
```

Comment:
Can you also test the other case (`replaceIfPresent` returns a `Future` completed with `false`)?